dist/session.js

```diff
@@ -157,7 +160,7 @@ session.delimiter = function (str) {
   if (str === undefined) {
     return this._delimiter;
   }
-  this._delimiter = String(str).trim() + ' ';
```

Comment:
I understand that sometimes we need to add fixes that as not necessarily intuitive enough. I tend to add comments around it anytime I have to do things like that.

This way it helps future developers working on the library.